src/lib/atoms/strava.ts

```
@@ -0,0 +1,165 @@
+    // Trigger workout matching logic
+    return Promise.resolve()
+  }
+)
+
+// Matching summary atom for workout comparison
+export const matchingSummaryAtom = atom<{
+  matched: number
+  unmatched: number
+  suggestions: number
+  byType?: Record<string, { matched: number; unmatched: number; suggestions: number }>
+  unmatchedWorkouts?: any[]
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)